frontend/src/pages/Landingpage.tsx

```diff
@@ -1,6 +1,17 @@
 import React from "react";
+import styled from '@emotion/styled';
+
+const Container = styled.div`
```

Comment:
I thought it was supposed to be this way, too, but this morning Nando made it work with const Whatever = styled('thingie')`, as opposed to styled.thingie`. So maybe this should be updated.